### DISCUSSION TITLE
Ability to set Search Bar separately from Toolbar

### DISCUSSION THREAD

**Discussion by alchem1no** [plateaukao/einkbro#323] [Ideas]
Hello! I have an idea: add an option to set the search bar separately from toolbar so search bar could be fully seen. So it will look something like this:

![Screenshot_20240114_121618_Browser.jpg](https://github.com/plateaukao/einkbro/assets/62944727/b5c00653-dfe2-423b-b57d-3ed1a427693e)

![Screenshot_20240114_121016_Yandex Start.jpg](https://github.com/plateaukao/einkbro/assets/62944727/a716630e-ecfd-4f0e-ad02-ba26b893f286)

**Comment by plateaukao** [comment]
will consider it. however, the UI needs a lot of effort to refactor, the probability is low.
